user_notification/tests.py

```diff
@@ -1,6 +1,8 @@
 from user_notification.models import JobAlert
 import pytest
 from datetime import datetime
+from accounts.models import UserAccount
```

Comment:
Import not needed unless you need to reference user details stored in UserAccount(birth date, gender or contact number)

Reply:
I edited this in the next PR, and deleted this import